client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-placeholder-dialog.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+
+class PaymentMethodStripePlaceholderDialog extends Component {
+	render() {
+		const { translate } = this.props;
+		const buttons = [
+			{ action: 'cancel', disabled: true, label: translate( 'Cancel' ), onClick: noop },
+		];
+
+		return (
+			<Dialog additionalClassNames="payments__dialog woocommerce" buttons={ buttons } isVisible>
```

Comment:
mixed class names `payments__ ... ` and `stripe__ ...` below.

Reply:
Yep - we use `payments__dialog` as an exception for all payment method dialogs